diff --git a/.env.example b/.env.example
new file mode 100644
index 0000000..cee619b
--- /dev/null
+++ b/.env.example
@@ -0,0 +1,17 @@
+PORT=3000
+AWS_REGION=ap-southeast-2
+COGNITO_REGION=ap-southeast-2
+COGNITO_USER_POOL_ID=
+COGNITO_APP_CLIENT_ID=
+COGNITO_APP_CLIENT_SECRET=
+# Names (ARNs) of SSM parameters created by Terraform
+PARAM_API_BASE_URL=
+PARAM_YT_API_BASE=
+PARAM_TMDB_API_BASE=
+PARAM_PIXABAY_API_BASE=
+# Secrets Manager identifiers created by Terraform
+SECRET_YT_API_KEY=
+SECRET_TMDB_API_KEY=
+SECRET_TMDB_V4_TOKEN=
+SECRET_PIXABAY_API_KEY=
+SECRET_DB_CREDENTIALS=
